source/server/proto_descriptors.cc

```diff
@@ -4,14 +4,15 @@
 #include "envoy/api/v2/eds.pb.h"
 #include "envoy/api/v2/lds.pb.h"
 #include "envoy/api/v2/rds.pb.h"
+#include "envoy/api/v2/srds.pb.h"
 #include "envoy/service/discovery/v2/ads.pb.h"
 #include "envoy/service/discovery/v2/hds.pb.h"
 #include "envoy/service/ratelimit/v2/rls.pb.h"
 
 #include "common/common/assert.h"
 #include "common/common/fmt.h"
 #include "common/config/protobuf_link_hacks.h"
-#include "common/protobuf/protobuf.h"
+#include "common/grpc/common.h"
```

Comment:
I don't think this is the right location either. This is server global vs. organized around config. Let's chat in person about this PR, as I think it might be faster.

Reply:
Done.